sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/changelog.md

```diff
@@ -3,6 +3,15 @@
 ## Current version
 ###  Feature Additions
 
+## Version 4.1.1
+###  Feature Additions
+- Add PostgreSql Sink
+- Add Partition settings for Oracle, SapTable
+- Add dedicated source and dataset for several relational sources including ODBC, MySql
+- Add Azure Data Explorer Command activity, which allows users executing Kusto Control Commands
+### Breaking Changes

```

Comment:
This is not a breaking change - you are adding new parameters to the constructor that have default values at the end, so  it won't break existing customers. Remove this line.

Reply:
This PR contains both my change and Wenbo's changes. The note under the Breaking Changes is Wenbo's note which is related to her changes in PostgreSql.
You can have a look at her original PR which I took and integrated into my PR: https://github.com/Azure/azure-sdk-for-net/pull/7012/commits/10fec6da7d8843aad9a9fc9ba0bcf48a889e4b34.